Formula/vulkan-loader.rb

```diff
@@ -0,0 +1,45 @@
+class VulkanLoader < Formula
+  desc "Vulkan ICD Loader"
+  homepage "https://github.com/KhronosGroup/Vulkan-Loader"
+  url "https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.220.tar.gz"
+  sha256 "fe5b65e5d88febb1220ae59de363ed8b2794f8c861d6d74efb14aecefd464559"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-Loader.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
+  depends_on "vulkan-headers"
+
+  on_linux do
+    depends_on "libx11"
+    depends_on "libxcb"
+    depends_on "libxrandr" => :build
+    depends_on "wayland"
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+                    "-DFALLBACK_DATA_DIRS=#{HOMEBREW_PREFIX}/share:/usr/local/share:/usr/share",
+                    *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    inreplace lib/"pkgconfig/vulkan.pc", /^Cflags: .*/, "Cflags: -I#{Formula["vulkan-headers"].opt_include}"
```

Comment:
Seems reasonable, though currently that just gets you a `-I`, which at best is useless and at worst would generate a compiler warning for the nonexistent directory. Maybe just hold off and consider doing something here if the package ever adds something? (I don't really expect it tbh, since this package really just provides the runtime lib rather than headers.)